docs/guide/dep-pre-bundling.md

```diff
@@ -50,15 +50,15 @@ Both `include` and `exclude` can be used to deal with this. If the dependency is
 
 ### File System Cache
 
-Vite caches the pre-bundled dependencies in `node_modules/.vite`. It determines whether it needs to re-run the pre-bundling step based on a few sources:
+Vite caches the pre-bundled dependencies in `cacheDir`. It determines whether it needs to re-run the pre-bundling step based on a few sources:
 
 - The `dependencies` list in your `package.json`
 - Package manager lockfiles, e.g. `package-lock.json`, `yarn.lock`, or `pnpm-lock.yaml`.
 - Relevant fields in your `vite.config.js`, if present.
 
 The pre-bundling step will only need to be re-run when one of the above has changed.
 
-If for some reason you want to force Vite to re-bundle deps, you can either start the dev server with the `--force` command line option, or manually delete the `node_modules/.vite` cache directory.
+If for some reason you want to force Vite to re-bundle deps, you can either start the dev server with the `--force` command line option, or manually delete the `cacheDir` cache directory.
```

Comment:
Probably it's better to leave them as-is, as people who changed `cacheDir` explicitly should already understand it. Otherwise, I am a bit concern that people who come to this page without the context will not aware that `cacheDir` is actually a config option and failed to find it.